integrated_channels/lms_utils.py

```diff
@@ -51,5 +62,25 @@ def get_single_user_grade(course_id, user):
     if not CourseGradeFactory:
```

Comment:
did you mean line 38 with this? 

```
user_cert = get_certificate_for_user(username=user.username, course_key=course_key)
```

checked that method again in edx-platform and indeed it expects username instead of user (odd)

Reply:
```
def get_certificate_for_user(username, course_key, format_results=True):
    """
    Retrieve certificate information for a particular user for a specific course.

    Arguments:
        username (unicode): The identifier of the user.
        course_key (CourseKey): A Course Key.
```

Reply:
Ew. not great but obviously outside the scope of this ticket and our team. Will close comment.